packages/flutter/lib/src/rendering/platform_view.dart

```diff
@@ -189,8 +206,8 @@ class RenderAndroidView extends RenderBox with _PlatformViewGestureMixin {
 
     // Clip the texture if it's going to paint out of the bounds of the renter box
     // (see comment in _paintTexture for an explanation of when this happens).
-    if (size.width < _currentAndroidViewSize.width || size.height < _currentAndroidViewSize.height) {
-      context.pushClipRect(true, offset, offset & size, _paintTexture);
+    if (size.width < _currentAndroidViewSize.width || size.height < _currentAndroidViewSize.height && clipBehavior != Clip.none) {
```

Comment:
`&&` has higher precedence than `||`

I have no idea what the logical flow was intended to be here, but it looks wrong.

Reply:
That does look wrong. Wanna send a PR to fix it?

Reply:
See https://github.com/flutter/flutter/issues/67343

Reply:
Agree it's wrong. Thanks for finding it!

Reply:
So sorry for this guys 😞